### DISCUSSION TITLE
翻译模型 README

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#380] [MMAction2]
# 翻译模型 README 

翻译 Swin & VideoMAE README: 
- 将 configs/recognition/swin/README.md 翻译为中文
- 将 configs/recognition/videomae 翻译为中文
- 可以参考 0.x 分支中其他模型的中文 README

## Task Information

**Repository**：MMAction2

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：Jiayi-Song-UF

**Claim Time**：2023年07月31日

**PR Link**：https://github.com/open-mmlab/mmaction2/pull/2621

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by Jiayi-Song-UF** [comment]
Claim the mission

**Comment by Jiayi-Song-UF** [comment]
https://github.com/open-mmlab/mmaction2/pull/2621

**Reply by crazysteeaam** [reply]
well done~
